### ISSUE TITLE
Add support for multiple backend page alerts

### ISSUE THREAD

**Issue by remie** [symphonycms/symphony-2#759]
After installing a Symphony I get an page alert saying I should remove the install / update script. Not doing so will result in this alert to remain visible. 

Next I tried to install the CDI extension, which has a check to see if you have made the required modifications to core files. If this is not the case, it will add a page warning.

I will not see this second alert because of the notification to remove the install / update scripts.

In short: Symphony should have a UI implementation to deal with multiple alert notifications.
